TestingHelpers/MockFileSystem.cs

```diff
@@ -50,40 +43,19 @@ public MockFileSystem(IDictionary<string, MockFileData> files, string currentDir
             }
         }
 
-        public FileBase File
-        {
-            get { return file; }
-        }
+        public FileBase File { get; }
 
-        public DirectoryBase Directory
-        {
-            get { return directory; }
-        }
+        public DirectoryBase Directory { get; }
 
-        public IFileInfoFactory FileInfo
-        {
-            get { return fileInfoFactory; }
-        }
+        public IFileInfoFactory FileInfo { get; }
 
-        public PathBase Path
-        {
-            get { return pathField; }
-        }
+        public PathBase Path { get; }
 
-        public IDirectoryInfoFactory DirectoryInfo
-        {
-            get { return directoryInfoFactory; }
-        }
+        public IDirectoryInfoFactory DirectoryInfo { get; }
 
-        public IDriveInfoFactory DriveInfo
-        {
-            get { return driveInfoFactory; }
-        }
-	
-        public IFileSystemWatcherFactory FileSystemWatcher
-        {
-            get { return fileSystemWatcherFactory; }
-        }
+        public IDriveInfoFactory DriveInfo { get; }
+
+        public IFileSystemWatcherFactory FileSystemWatcher { get; }
 
         public PathVerifier PathVerifier
```

Comment:
This could be
```csharp
public PathVerifier PathVerifier => pathVerifier;
```

to make it a bit more consistent with the new getters. Do we care about the newline between each property?

Reply:
Fixed! I'm not too familiar with the `[NonSerialized]` attribute, but could we drop the private readonly variable entirely and have:
```
[NonSerialized]
public PathVerifier PathVerifier { get; }
```

I personally like the newline between the properties, but I prefer consistency to any personal preference I have :)

Reply:
Not a huge deal just to leave it as going to be a matter of preference.. and we don't have any standards at the moment.